src/tv2-common/evaluateCues.ts

```diff
@@ -210,7 +212,7 @@ export async function EvaluateCuesBase(
 
 	for (const cue of cues) {
 		if (cue && !SkipCue(cue, options.selectedCueTypes, options.excludeAdlibs, options.adlibsOnly)) {
-			const shouldAdlib = /* config.showStyle.IsOfftube || */ options.adlib ? true : cue.adlib ? true : false
+			const shouldAdlib = !!(options.adlib || cue.adlib)
```

Comment:
Is the double negation necessary?

Reply:
`cue.adlib` is optional, and later we're passing `shouldAdlib` to `EvaluateCueClearGrafiks`, which expects a boolean. Either here or there we need double negation.